samples/nrf9160/aws_iot/README.rst

```diff
@@ -0,0 +1,153 @@
+.. _aws_iot:
+
+nRF9160: AWS IoT
+################
+
+The AWS IoT sample shows the communication of an nRF9160-based device with the AWS IoT message broker over MQTT.
+This sample uses the :ref:`lib_aws_iot` library.
+
+Overview
+********
+
+The sample publishes two different types of messages in `JSON`_ object string format to the AWS IoT message broker.
+These messages are addressed to the AWS IoT shadow topic ``$aws/things/<thingname>/shadow/update``.
```

Comment:
```suggestion
These messages are sent to the AWS IoT shadow topic ``$aws/things/<thingname>/shadow/update``.
```

Consider using `sent`, because in MQTT addresses are not used but topics.